app/models/course_enrichment.rb

```diff
@@ -110,6 +110,15 @@ def unpublish(initial_draft: true)
     update(data)
   end
 
+  def copy_to_course(new_course)
+    new_enrichment = self.dup
+    if new_enrichment.published?
+      new_enrichment.last_published_timestamp_utc = nil
```

Comment:
you want to nil it for draft as well

a draft with last published is a subsequent draft

this will most likely lead to frontend displaying it wrong

Reply:
Done.